Formula/samba.rb

```diff
@@ -0,0 +1,123 @@
+class Samba < Formula
+  # Samba can be used to share directories with the guest in QEMU user-mode (SLIRP) networking
+  # with the `-net nic -net user,smb=/share/this/with/guest` option.
+  # The shared folder appears in the guest as "\\10.0.2.4\qemu".
+  desc "SMB/CIFS file, print, and login server for UNIX"
+  homepage "https://samba.org/"
+  url "https://download.samba.org/pub/samba/stable/samba-4.14.5.tar.gz"
+  sha256 "bb6ef5d2f16b85288d823578abc453d9a80514c42e5a2ea2c4e3c60dc42335c3"
+  license "GPL-3.0-or-later"
+
+  # configure requires python3 binary to be present, even when --disable-python is set.
+  depends_on "python@3.9" => :build
+  depends_on "gnutls"
+
+  uses_from_macos "perl"
+
+  resource "Parse::Yapp" do
+    url "https://cpan.metacpan.org/authors/id/W/WB/WBRASWELL/Parse-Yapp-1.21.tar.gz"
+    sha256 "3810e998308fba2e0f4f26043035032b027ce51ce5c8a52a8b8e340ca65f13e5"
+  end
+
+  def install
+    # avoid `perl module "Parse::Yapp::Driver" not found` error on macOS 10.xx (not required on 11)
+    if MacOS.version < :big_sur
+      ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+      ENV.prepend_path "PATH", libexec/"bin"
+      resource("Parse::Yapp").stage do
```

Comment:
Is this resource needed at runtime? How does `samba` know where to find it at runtime, if it's needed then?

Reply:
It's been a long time since I looked at samba but I'm pretty sure it's only needed for the build and not required at run time.

Reply:
If it's only needed at build time, install it into `buildpath` rather than `libexec`.

Reply:
fixed